crates/router/src/connector/rapyd.rs

```diff
@@ -483,7 +493,7 @@ impl
         &self,
         req: &types::PaymentsCaptureRouterData,
     ) -> CustomResult<Option<types::RequestBody>, errors::ConnectorError> {
-        let req_obj = rapyd::CaptureRequest::try_from(req)?;
+        let req_obj = rapyd::CaptureRequest::try_from(&req)?;
         let rapyd_req = types::RequestBody::log_and_get_request_body(
```

Comment:
Hey @shivansh-bhatnagar18 
You have to do the transformation here as well in Capture TryFrom.

You have to do it for `req.request.amount_to_capture`

Reply:
Ohh Thank You for pointing out my mistake. I have made the changes and will be creating a draft PR for the checks.
After these checks I'll create a PR

Reply:
This looks good to me, resolving this comment